web3/manager.py

```diff
@@ -102,6 +98,26 @@ def _make_request(self, method, params):
                 )
             )
 
+    async def _make_async_request(self, method, params):
```

Comment:
Naming convention for this type of thing has been to use a `coro_` prefix so maybe `_coro_make_request`.